tests/integration/utils.py

```diff
@@ -264,40 +264,44 @@ def assert_element_from_response(response_data, expected_element, element_name):
         assert response_element == expected_element
 
 
-def assert_expected_files(source_path, expected_file_urls=None, check_content=True):
+def assert_expected_files(source_path, expected_files):
```

Comment:
I wonder if we want to add git submodules to a `data` directory instead of downloading these files individually. Then we'd have them locally and we could just compare the whole directories instead of listing all the expected files in the configuration files.

Thoughts?

Reply:
On a second thought, having git references for test repositories attached to our git history may be a bad idea. We could still fetch the complete archive for the test repository instead of doing it in a per-file basis.

Reply:
@athos-ribeiro I like your second idea. I just not sure how to realize it. 
The archive from Cachito contains `app` and `deps` directories. So should we keep expected archives in [cachito-testing](https://github.com/cachito-testing/) organization? Download them in `assert_expected_files` and check binary equality with the response from Cachito?

Reply:
The app directory will contain the files in a given revision for a repository. The deps directory will contain the files for the specific dependencies (this will depend on where the dependency was fetched from, but we seem to be skipping file checks for these anyway)